nuvlaedge/agent/agent.py

```diff
@@ -5,7 +5,9 @@
 
 import logging
 import os
+
 from copy import copy
+from collections.abc import Callable
```

Comment:
Why import Callable from collections? I think the root package containing it is typing. However, since we moved to python 3.11 it should come in python as the type `callable`

Reply:
Because the doc say `typing.Callable` is a deprecated alias of `collections.abc.Callable` (since python 3.9).
https://docs.python.org/3.11/library/typing.html#typing.Callable
I did search on the doc how to use `callable` but I haven't found anything.